src/applications/virtual-agent/components/chatbox/Chatbox.jsx

```diff
@@ -130,8 +134,18 @@ export default function Chatbox(props) {
 }
 
 function App(props) {
+  const { virtualAgentWebchatUpgrade } = useSelector(
+    state => {
+      return {
+        virtualAgentWebchatUpgrade:
+          state.featureToggles[FEATURE_FLAG_NAMES.virtualAgentWebchatUpgrade],
+      };
```

Comment:
I once had to check if the feature flags were even working at some point (because they weren't at the time of deploy, something no one saw coming). 

Wouldn't hurt to use some optional chaining here on state?.featureToggles.FEATURE_FLAG_NAMES.virtualAgentWebchatUpgrade